src/js/app/controllers/community/JoinCommunityByUrlCtrl.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function($scope, $controller, community, code) {
+  'ngInject';
+
+  $scope.community = community;
+  $scope.code = code;
+
+  $controller('JoinCommunityCtrl', {$scope: $scope})
```

Comment:
This puts all the variables and functions from JoinCommunityCtrl onto $scope. It's like subclassing JoinCommunityByUrlCtrl from JoinCommunityCtrl.